coffee_lib/src/types/mod.rs

```diff
@@ -3,6 +3,11 @@ use serde::{Deserialize, Serialize};
 
 use crate::plugin::Plugin;
 
+#[derive(Serialize, Deserialize)]
+pub struct CoffeeRemove {
+    pub plugin: Option<Plugin>,
```

Comment:
```suggestion
    pub plugin: Plugin,
```

there is some case where the plugin in null in the json?

Reply:
No there isn't.
We already check if the plugin exists once.
I will fix this